Roadmap.md

```diff
@@ -0,0 +1,81 @@
+    - `[sol-compiler](https://sol-compiler.com)`: Solidity compilation
+
+    We plan to prioritize `sol-trace` and to look into the benefits that the others tools could have on the developer experience and decide whether to work on that *short-term*.
+
+- Identify contracts like the `MiniMeTokenFactory` which do not need to be deployed every time, but rather reused, and publish them to `aragonpm.eth` (just like the `dao-kits`)
+
+- Wizard onboarding 🧙‍♂️
+    - The goal of this tool will be to help new users create a custom DAO
+    - "Import" the onboarding of the multisig and democracy kits from the ui client
+    - Improve UX for installing "core" apps (e.g.: TokenManager, Voting, etc.), because currently is a bit tricky as you have to check what arguments the `initialize` function takes from the app's contract and pass them with `--app-init-args`, e.g.: `dao install [dao-address] voting --app-init-args 0x00000001 600000000000000000 250000000000000000 604800`. We should do this, instead of the users, so they can see what are the needed parameters with `--help`  or `--interactive`. For example `dao install voting --help` could return `dao install voting [dao-address] [token-address] [support-required] [quorum] [vote-time]`
+    - Commands can become interactive (with prompts) using `[yargs-interactive](https://www.npmjs.com/package/yargs-interactive)`
+
+    Example: To create a new MiniMe token, you would use the `dao token new` command, which takes the following arguments `<token-name> <symbol> [decimal-units] [transfer-enabled]`, e.g.: `dao token new "Aragon Network Token" "ANT" 18 true`. Using
+
+    `dao token new --interactive` though, would output this:
+    
+    ![alt text](https://s3.us-west-2.amazonaws.com/secure.notion-static.com/ba2b8fd3-400b-425e-8d71-59381a0f76d8/Untitled.png?AWSAccessKeyId=ASIAT73L2G45MUQUGMXG&Expires=1549812298&Signature=jzG2sPDgeYcB9Zd9varZlqIRzF4%3D&x-amz-security-token=FQoGZXIvYXdzEB0aDKFd0gNValJVyxjuWSK3Ax0SjqEaNpRBlQtp9%2BSAGi1RNhFEpHI9dkZx%2BJA2%2BgdHr7z4IMAQRGHemmf28rttDLHuyQlvXVmNwF6OLMCl2sYyPsZCjmm2yDw5W8FSighcYlvushqFJKDTWlhMz%2Fzy3heWBDhH59obwLk3gibWiKvbyNkwtEPR2CKOIBQ1CryNgbjxqw9yyhX2JDUkQqQ7JToESHZi3xMKFvpbyS3PoQfchaC9jjZ7weDJ1xh4Db9qu7CEJm8wGohxRcIegtL56O7VUmoCZzNKnDoODZsllIqFS64HOKTOOPOyXfFkf4WDyTNZKpqihJ9xaes4Lv1WujW5rIl5vd0sO%2FPrdWKHdNvze6CbLVsLH%2FabwdM3R8FuYwuR7SQqcDZCDRxVgb7DQzFR%2F0KC%2BURzVPcsLbTp9HQYA4XmDWDYH5uIvTmVZoCVaDEnGSCka7j7Oiv%2FG7ewkv7w0U1lhAkBw%2FEBaRDSqMSyXIeuz%2Be3Qu6k6BZvQBlom%2BHMFGkYpwawkQ7iZ6pxMvmC5UjTIB84XPvmhup2khU1sSbHOSUX6fsp6Xpyqfl%2BbKW5FwbVoIItg6dyY44NykiZDqAVLLko7%2Fv64gU%3D)
+    
+## Ongoing
+
+- Active on the different dev-related aragon.chat channels. We aim to respond within one day and to allocate ~1 hour each day to debug issues like [this](https://github.com/aragon/aragon-react-kit-boilerplate/issues/19), help people deploy to testnets, etc.
+- Aragen maintenance
+- Prioritize this roadmap
```

Comment:
**Brett:** My thinking on priorities:

1. Set up some smoke tests with E2E testing (1-2 weeks)
2. Short term bug fixes and items (2-4 weeks)
3. Docs restructuring (1 week)
4. Monorepo refactor?
5. Interactive wizards
6. Transition to TS
7. Other long-term items

May also be good to speak with Autark to see if they have any lingering pains with the CLI (there's one issue with stopping IPFS that's solved by sleeping, but would be good if we didn't end the process until we knew that IPFS was stopped). I don't foresee too many items that A1 will need in the next two to three months which Luke hasn't already brought up.

Reply:
**Daniel:** Thanks a lot for the feedback!!
Is there an issue for that ipfs behavior?
What do you think about open PRs?
@sohkai

Reply:
@0x6431346e There's no issue for it yet, but can be easily tested by going between this commit and the one before it: https://github.com/AutarkLabs/planning-suite/commit/e3178ebd51c4ac948d7fb06ea4609fe78d506c7d.

As for open PRs, hopefully those can go with 1 and 2 :).